background/get-addon-settings.js

```diff
@@ -59,57 +59,11 @@ chrome.storage.sync.get(["addonSettings", "addonsEnabled"], ({ addonSettings = {
         addonsEnabled["block-count"] = true;
       }
       if (addonId === "editor-dark-mode") {
-        // Transition v1.22 to v1.23
-        // TurboWarp Dark preset changes:
-        updatePresetIfMatching(
-          manifest.presets.find((p) => p.id === "tw-dark"),
-          settings,
-          {
-            page: "#111111",
-            primary: "#ff4d4d",
-            highlightText: "#ff4d4d",
-            menuBar: "#333333",
-            activeTab: "#1e1e1e",
-            tab: "#2e2e2e",
-            selector: "#1e1e1e",
-            selector2: "#2e2e2e",
-            selectorSelection: "#111111",
-            accent: "#111111",
-            input: "#1e1e1e",
-            workspace: "#1e1e1e",
-            categoryMenu: "#111111",
-            palette: "#111111",
-            fullscreen: "#111111",
-            stageHeader: "#111111",
-            border: "#ffffff0d",
-          },
-          1
-        );
-        // Experimental Dark changes:
-        updatePresetIfMatching(
-          manifest.presets.find((p) => p.id === "experimentalDark"),
-          settings,
-          {
-            page: "#001533",
-            primary: "#4d97ff",
-            highlightText: "#4d97ff",
-            menuBar: "#4d97ff",
-            activeTab: "#282828",
-            tab: "#192f4d",
-            selector: "#030b16",
-            selector2: "#192f4d",
-            selectorSelection: "#282828",
-            accent: "#282828",
-            input: "#282828",
-            workspace: "#282828",
-            categoryMenu: "#282828",
-            palette: "#333333",
-            fullscreen: "#282828",
-            stageHeader: "#333333",
-            border: "#444444",
-          },
-          2
-        );
+        // Transition v1.27 to v1.28
+        if (settings.palette !== undefined && settings.palette.length === 7) {
+          settings.palette += "cc";
```

Comment:
Could we have a comment to describe this, because appending "cc" to the string isn't very clear here? it makes sense in the context of this pr but not for someone looking at this file by itself.